cogs/logging/logging.py

```diff
@@ -179,6 +179,28 @@ async def timezone_set(self, ctx, *, timezone: zoneinfo.ZoneInfo):
         await Timezones.insert(update_on_conflict=Timezones.timezone, user_id=ctx.author.id, timezone=timezone.key)
         await ctx.tick()
 
+    @timezone.command(name='get')
+    async def timezone_get(self, ctx, *, user: discord.Member):
+        """Get a member's timezone."""
+        timezone = await Timezones.fetchrow(user_id=user.id)
+        
+        if timezone is None:
+            embed = discord.Embed(
+                title = "Error",
+                description = f"User {user.name} doens't have a saved timezone.",
+                colour = discord.Colour.red()
+            )
+            return await ctx.send(embed = embed)
```

Comment:
Use built-in error handler.

```suggestion
        if timezone is None:
           raise commands.BadArgument(f"User {user.name} doesn't have a saved timezone.")
```